src/serving.ts

```diff
@@ -213,7 +215,9 @@ export class Server {
     let requestData: AnalyzeCommentData = req.body as AnalyzeCommentData;
 
     let requestedAttributes: RequestedAttributes = {};
-    requestedAttributes[this.config.toxicityAttribute] = {
+    const attribute =
+      requestData.modelName ? requestData.modelName : this.config.toxicityAttribute;
```

Comment:
NITs:
* You could shorten this to `const attribute = requestData.modelName || this.config.toxicityAttribute`
* Since this is duplicated above, you could consider refactoring to a function like `getRequestedAttribute(requestData)`, at least if this ever grows more complex